pkg/bootstrap/api/types.go

```diff
@@ -95,3 +95,6 @@ const (
 	// tokens (in addition to any groups from BootstrapTokenExtraGroupsKey).
 	BootstrapDefaultGroup = "system:bootstrappers"
 )
+
+// DefaultTokenUsages specifies the default functions a token will get.
+var DefaultTokenUsages = []string{"signing", "authentication"}
```

Comment:
I'd call this `KnownTokenUsages` instead. This package doesn't assume defaults for how kubeadm should create the tokens.

Reply:
Reasonable! :+1: 
updated.